omnibus.rb

```diff
@@ -24,6 +24,10 @@
 # package_dir   './local/omnibus/pkg'
 # package_tmp   './local/omnibus/pkg-tmp'
 
+# Windows architecture defaults - set to x86 unless otherwise specified.
+# ------------------------------
+windows_arch   ENV['OMNIBUS_WINDOWS_ARCH'] || :x86
```

Comment:
Curious why we're setting an environment variable here. Just so we can do one-off builds locally w/ 64 bit? Also, environment variables will always be strings, but the default is a symbol. Probably won't matter but would be more consistent for the default to be "x86".